Training_Preparation.md

```diff
@@ -0,0 +1,275 @@
+
+# Training
+
+Today's training consists of the following excercises:
+
+1. Simple Interaction with the Digital Twin Registry
+2. Creating a New Shell Descriptor
+3. Interaction between EDC and DTR
+4. Creating a New Sample Asset on Our EDC
+5. Inserting the Sample Asset API into a Shell Descriptor 
+6. Testing Our Sample Asset API
+6. Create your own ShellDescriptor with that asset in Part Type lifecycle following [Industry Core KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Industry%20Core%20Kit/Software%20Development%20View/Digital%20Twins%20Development%20View%20Industry%20Core%20Kit#property-specificassetids) and [Digital Twin KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/dt-kit-software-development-view)
+7. Verify that it works with a query following same approach as in 4.
+
+## Prerequisites
+
+- VM with Linux like Ubuntu, 8 GB of RAM available on that VM  
+- docker installed on VM (Get it [here](https://docs.docker.com/engine/install/ubuntu/) )  
+- 'jq' installed on VM (Get it [here](https://jqlang.github.io/jq/download/) )  
+- postman installed on VM (Get it [here](https://learning.postman.com/docs/getting-started/installation/installation-and-updates/) )  
+- git installed on VM (usually pre-installed on most linux systems)  
+- a code editor for simple editings in JSON ([VS Code](https://code.visualstudio.com/docs/setup/linux) recommended)
+
+
+## Getting Started
+
+- clone the PURIS project from the FraunhoferISST github [repository](https://github.com/FraunhoferISST/puris)  
+
+- move your shell into the folder 'puris'  
+
+
+- ```git switch feat/training```   
+
+
+- Note: The above-mentioned branch is not present, if you mistakenly cloned puris from the tractusX-repository!
+
+
+- move your shell into the folder 'puris/local'  
+
+
+- ```sh launch_infrastructure.sh```  
+
+
+
+Wait for the shell-script to finish (should take 1 or 2 minutes)  
+
+
+
+Now, again from the 'puris/local' folder do: 
+
+ ```docker compose up```  
+
+
+
+This should take another 1 or 2 minutes. In the meantime, please start your Postman and [import](https://learning.postman.com/docs/getting-started/importing-and-exporting/importing-data/) the 'TX-Training' collection from the 'puris/local/postman' folder 
```

Comment:
```suggestion
- open a terminal and clone the PURIS project from the FraunhoferISST GitHub [repository](https://github.com/FraunhoferISST/puris) (`git clone` command)  

- after successful cloning, move your shell into the folder 'puris' folder (`cd puris`)

- Switch the branch to the training: `git switch feat/training` 
  - _Note: The above-mentioned branch is not present if you mistakenly cloned puris from the eclipse-tractusx repository!_

- move your shell into the folder 'local' (`cd local`) 

- Launch the infrastructure with `sh launch_infrastructure.sh`


Wait for the shell script to finish (it should take around 1 or 2 minutes, depending on your hardware)

Now, again from the `puris/local` folder, run the docker-compose setup with `docker compose up`

This should take another 1 or 2 minutes. In the meantime, please start your Postman and [import](https://learning.postman.com/docs/getting-started/importing-and-exporting/importing-data/) the `TX-Training` collection from the `puris/local/postman` folder 
```

Reply:
Tom had already covered that part in his remarks and I rewrote most of it.